packages/castmill/lib/castmill_web/endpoint.ex

```diff
@@ -80,9 +80,16 @@ defmodule CastmillWeb.Endpoint do
   plug(Plug.Session, @session_options)
   plug(CastmillWeb.Router)
 
+  # Origin of different players
```

Comment:
When the app is running on a hardware it is served from some kind of localhost web server running on that device. Any http(s) request that the app makes will have that host as origin and in order to get CORS to work, that origin must be among the allowed origins of the castmill server.

- An android app serves the local files from https://localhost so we have to have that origin among the allowed origins in our backend endpoints.
- Webos serves from file://com.lg.app.signage so that origin has to be added as well.
- Electron apps serve the local assets from http://localhost:5173
- Regarding the castmill legacy adapter the matter is slightly different. It will be served on the old castmill player url and act as an adapter between castmill legacy and castmill2 so its origin will be https://player.castmill.com, https://player.castmill.io or similar. These urls will have to be added to the allowed origins for castmill legacy to work. This is however, only relevant to us and shouldn't be hard coded.

To be able to handle this without hard coded origins, we probably need some way to configure multiple allowed origins per network. Currently we can only configuer one as far as I know.